service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ConditionLinkSetMapper.java

```diff
@@ -203,4 +207,11 @@ private boolean filterOutListResourceType(Reference reference) {
     private boolean checkIfReferenceIsObservation(Reference reference) {
         return reference.getReferenceElement().getResourceType().equals(ResourceType.Observation.name());
     }
+
+    private String buildCode(Condition condition) {
+        if (condition.hasCode()) {
+            return codeableConceptCdMapper.mapCodeableConceptToCd(condition.getCode());
+        }
+        throw new EhrMapperException("Condition code not present");
```

Comment:
Can we get a test for the exception scenario too?

Reply:
done